README.md

```diff
@@ -232,6 +247,55 @@ namespace/name is used as the OAEP input parameter, ensuring that the
 The generated `Secret` is marked as "owned" by the `SealedSecret` and
 will be garbage collected if the `SealedSecret` is deleted.
 
+After sime time, the key used to encrypt a `SealedSecret` is replaced by
+a newer key. The `SealedSecret` can still be decrypted by the old key,
+but it is recommended to update with the latest key. This can be a
+problem without access to the private key outside the cluster.
+
+The `SealedSecret` can be re-encrypted by the cluster using an HTTP POST
+to `/v1/rotate`.
+
+**WARNING**
+A current issue in serializing the data prevents the `/v1/rotate` endpoint
+from returning a valid `SealedSecret` object. A workaround exists in the
+`./kubeseal --rotate` flag.
+
+### Blacklisting
+Sealed secrets provides a blacklisting feature to warn users of secrets
+that may have been encrypted with a compromised private key. The
+controller maintains a list of key names corresponding to keys that are
+suspected to be compromised. This list is also stored as a secret in
+the controllers namespace and maintained similarly to the key list
+itself.
+
+The `SealedSecret`s controller will refuse to work with blacklisted
```

Comment:
I originally thought to keep them around in the case the original secrets are lost somehow. A blacklisted private key can still be used to un-encrypt it if need be. I'm not sure how useful that actually is though.

Reply:
I could imagine for key-rotation that it would be useful to "deprecate" a key - still use it for decryption but try to prevent it being used for new encryption (hide it from --fetch-cert output, etc).

I think I want a clear workflow use-case here to make sure the extra code is actually helping someone.  Until then, I suggest we remove the blacklist code/feature until someone asks for it (we can go from nothing->something more easily than we can go something->something_else once it's a public released feature).

Reply:
@anzboi I'd love to reopen the conversation about this point if needed; let's proceed piecewise, this contribution looks already useful in the current shape; merging